src/client/components/modals/NotificationInfo/strategies/teams.tsx

```diff
@@ -0,0 +1,20 @@
+import { Form, Input } from 'antd';
+import React from 'react';
+import { useTranslation } from '@i18next-toolkit/react';
+
+export const NotificationTeams: React.FC = React.memo(() => {
+  const { t } = useTranslation();
+
+  return (
+    <>
+      <Form.Item
+        label={t('Teams Webhook URL')}
+        name={['payload', 'teamsWebhookUrl']}
+        rules={[{ required: true }]}
+      >
+        <Input placeholder={t('For example: https://example.com/callback')} />
+      </Form.Item>
```

Comment:
i like it, did you mind leave some document link about where can we add it? 

Reference: `src/client/components/modals/NotificationInfo/strategies/feishu.tsx`